src/Symfony/Component/Validator/Resources/translations/validators.id.xlf

```diff
@@ -36,11 +36,11 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>Bidang ini tidak diharapkan.</target>
```

Comment:
Google Translate says that `Bidang` translates as `field` and `Ruas` as `segment`, so I'd keep the original translation.